src/main/java/talker/command/MarkCommand.java

```diff
@@ -0,0 +1,32 @@
+package talker.command;
+
+import talker.Storage;
+import talker.TalkerException;
+import talker.Ui;
+import talker.task.TaskList;
+
+/**
+ * Represents mark command to mark task as complete
+ */
+public class MarkCommand extends Command {
+    // string representing task number to be marked
+    private String[] parsed;
```

Comment:
Parsed field could have been better named using a plural noun.
```suggestion
    private String[] parsedTokens;
```